client/src/main/python/slipstream/executors/node/NodeDeploymentExecutor.py

```diff
@@ -144,8 +144,9 @@ def onSendingReports(self):
                              verboseThreshold=util.VERBOSE_LEVEL_QUIET)
 
     def _execute_report_target_and_send_reports(self):
+        exports = {'SLIPSTREAM_REPORT_DIR': util.get_platform_reports_dir()}
         try:
-            self._execute_target('report', ssdisplay=False, ignore_abort=True)
+            self._execute_target('report', exports=exports, ssdisplay=False, ignore_abort=True)
```

Comment:
If you send `exports=None` to Popen, it will add all env variables of the parent process (the slipstream node executor).
But if you send something other than `None` this behaviour doesn't apply so in this case `SLIPSTREAM_REPORT_DIR` will be the only environment variable for the user script.
So I would add after the line 147 the following line:
```python
exports.update(os.environ)
```
Because of that the user script will probably fail.